polypomo

```diff
@@ -133,6 +138,57 @@ class Status:
             self.timer = Timer(self.worktime)
 
 
+@contextmanager
+def setup_conn():
+    global DATABASE_PATH
+    # check if the database exist
+    if (os.path.exists(DATABASE_PATH)):
+        print('database already exist')
+        conn = sqlite3.connect('polypomoDB.db')
+        cur = conn.cursor()
+        try:
+            yield cur
+        finally:
+            conn.commit()
+            conn.close()
+    else:
+        print('initiate database')
+        conn = sqlite3.connect('polypomoDB.db')
+        cur = conn.cursor()
+        cur.execute('''CREATE TABLE data
+               (days text, worked_time text)''')
+        conn.commit()
+        try:
+            yield cur
+        finally:
+            conn.commit()
+            conn.close()
+
+def db_save(time, status):
+    global START_TIME
+    if (status.status == 'work'):
+        if  not status.active and START_TIME != 0:
+            with setup_conn() as conn:
+                today = date.today()
+                worked_time = (time - START_TIME) / 60 / 60
+                todaysrow = conn.execute(f'SELECT * FROM data WHERE days = "{today}"').fetchall()
```

Comment:
Can you use SQL arguments here instead of string replacing variables?
See https://stackoverflow.com/a/902417
Same for other SQL statements below